rasa/model.py

```diff
@@ -374,6 +376,8 @@ def _get_fingerprint_of_config_without_epochs(
             for p in copied_config[key]:
                 if "epochs" in p:
                     del p["epochs"]
+                # if COMPONENT_INDEX in p:  # ToDo: Remove comment
```

Comment:
Here is another option: Don't take the component indices into account when generating the fingerprints.

Reply:
With this all tests under `test_train` succeed.